scheduler/stack.go

```diff
@@ -8,23 +8,10 @@ import (
 )
 
 const (
-	// serviceJobAntiAffinityPenalty is the penalty applied
-	// to the score for placing an alloc on a node that
-	// already has an alloc for this job.
-	serviceJobAntiAffinityPenalty = 20.0
-
-	// batchJobAntiAffinityPenalty is the same as the
-	// serviceJobAntiAffinityPenalty but for batch type jobs.
-	batchJobAntiAffinityPenalty = 10.0
-
-	// previousFailedAllocNodePenalty is a scoring penalty for nodes
-	// that a failed allocation was previously run on
-	previousFailedAllocNodePenalty = 50.0
-
 	// skipScoreThreshold is a threshold used in the limit iterator to skip nodes
-	// that have a score lower than this. -10 is the highest possible score for a
-	// node with penalty (based on batchJobAntiAffinityPenalty)
-	skipScoreThreshold = -10.0
+	// that have a score lower than this. -1 is the lowest possible score for a
+	// node with penalties (based on job anti affinity and node rescheduling penalties
+	skipScoreThreshold = 0.0
```

Comment:
comment is missing a closing parenthesis, but I'm curious why this changed from being equal to the value in the comment to 0. Seems strange for the comment to mention one number and the const is set to another.